### DISCUSSION TITLE
Date and time get/set

### DISCUSSION THREAD

**Discussion by piottolorenzo** [gicamm/homeassistant-comelit#59] [General]
Hi. Maybe a dumb idea. 
I'd love if possible to poll and set date and time on my comelit hub. 

The main issue is that I cannot make the hub automatically get the time from and NTP server and this is an issue when the main power triggers. I don't know the why but I think it is related to the network settings in the hub, which I cannot touch too much as the external network interconnects all the other hubs in my building. Using home assistant as time manager would be helpful

**Comment by gicamm** [comment]
Hi,
I don't know if the hub allows you to set the time using MQTT.
If so, it might be possible. 
However, you could set up your own NTP server. It's never a bad idea to use a local NTP server.
